src/SourceBlock/SourceBlock.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
will this change given your new layout? Mock?

Reply:
Yes this might change, but not too much. I built these components with the ability to be moved around so I'm hoping that there won't be too many changes